etcdserver/api/v3rpc/watch.go

```diff
@@ -42,12 +43,20 @@ func NewWatchServer(s *etcdserver.EtcdServer) pb.WatchServer {
 }
 
 var (
-	// expose for testing purpose. External test can change this to a
-	// small value to finish fast. The type is int32 instead of time.Duration
-	// in order to placate the race detector by setting the value with atomic stores.
-	ProgressReportIntervalMilliseconds = int32(10 * 60 * 1000) // 10 minutes
+	// External test can read this with GetProgressReportIntervalMilliseconds()
+	// and change this to a small value to finish fast with
+	// SetProgressReportIntervalMilliseconds().
+	progressReportIntervalMilliseconds = int32(10 * 60 * 1000) // 10 minutes
```

Comment:
This was changed to Milliseconds for atomics to avoid needing an explicit lock, but now it doesn't make much sense if there's going to be full functions to Get/Set.

Change to `progressReportInterval = 10 * time.Minute`?